index.html

```diff
@@ -4,18 +4,32 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>webdew</title>
+    <style>
+      .firstDiv{
+        font-family: 'Courier New', Courier, monospace;
+        padding: 0 20px;
+        background-color: bisque;
+      }
+      h1{
+        text-align: center;
+        text-decoration: underline;
+        background-color: black;
+        color: beige;
+        padding: 15px;
+      }
+    </style>
```

Comment:
good that you added style. 
```suggestion
    <style>
      .firstDiv{
        font-family: 'Courier New', Courier, monospace;
        padding: 0 20px;
        background-color: bisque;
      }
      h1{
        text-align: center;
        text-decoration: underline;
        background-color: black;
        color: beige;
        padding: 15px;
      }
    </style>
```
**but remember that you should create a css file for styling.**